src/api/webhook.ts

```diff
@@ -23,16 +23,15 @@ const WEBHOOKS_ACTIONS: { [key: string]: WebhookFunction } = {
 };
 
 export const incomingWebhook = async (request: Request, response: Response) => {
-    const webhookRequest: WebhookRequest<any> = request.body.values;
-    const context: AppContext = request.body.context;
+    const webhookRequest: WebhookAppCallRequest<any> = request.body;
```

Comment:
Can we provide something more meaningful than `any` here?

Reply:
This was solved in commit [d91fc61](https://github.com/liz-segura98/mattermost-app-opsgenie/pull/1/commits/d91fc61f3d5c4e7a5418a74cce2c5a72c991a951)